### PR TITLE
finished javascript week1

### PR DESCRIPTION
**Pull request by SeonaV**

### CODE REVIEW

File: javascript/javascript1/week1/myjavascript.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+// Peter house 
+
+const gardenSizeInM2 = 100 ;
+ const PWidth = 8;
+ const PHeight = 10;
+ const PDepth = 10;
+
+ //Calculation for Volume in Meters for Peter
+
+ let PvolumeInMeters = PWidth * PHeight * PDepth;
```
Comment (asmafassy) [inline javascript/javascript1/week1/myjavascript.js]: Why let? :)

File: javascript/javascript1/week1/myjavascript.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+ //Peter House Price
+
+ const actualPeterPrice = 2500000;
+
+ housePriceforPeter = PvolumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+
+ if (housePriceforPeter < actualPeterPrice){
+     console.log("Peter High price")
```
Comment (asmafassy) [inline javascript/javascript1/week1/myjavascript.js:45]: What do you think if the amount he paid is less than the actual amount?
 according to the condition statement isn't correct.
 
 He would have profited from it some amount and you can find that too in here, give it a try!! :see_no_evil:

File: javascript/javascript1/week1/myjavascript.js [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+ } else if (housePriceforPeter == actualPeterPrice){
```
Comment (asmafassy) [inline javascript/javascript1/week1/myjavascript.js]: You can use "==="  strict equality operator in order to receive the error-free results.

File: javascript/javascript1/week1/myjavascript.js [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+ housePriceforJulia = JvolumeInMeters  * 2.5 * 1000 + JgardenSizeInM2 * 300;
+
+ if (housePriceforJulia < actualJuliaPrice){
+     console.log("Julia high pricd")
+ } else if (housePriceforJulia == actualJuliaPrice){
+    console.log("Julia expected amount")
+ } else {
+    console.log("Julia low price")
+ };
```
Comment (asmafassy) [inline javascript/javascript1/week1/myjavascript.js:73]: You have done a great job writing the conditions and the statements in Peter's and Julia's cases.
Also I can see the same error here too. :) I hope you would have figured it out by now. :technologist:

File: javascript/javascript1/week1/myjavascript.js [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+  const secondWords = ['Industry', 'Collaborations','Information technology', 'Finance', 'Trading', 'Technology','Artificial Technology','Consultation','Avition', 'Oil & Refineries']
+
+
+ //Random name
+ const randomFirstWords = firstWords[ Math.floor(Math.random() * 10)];
+ const randomSecondWords = secondWords [Math.floor(Math.random() * 10)];
+
+ let startupName = randomFirstWords + ' ' + randomSecondWords;
+
+ console.log("The startup: " + startupName + " contains " + startupName.length + " characters");
```
Comment (asmafassy) [inline javascript/javascript1/week1/myjavascript.js:88]: Well done :sparkles: You have done a great job.  :blossom:

**Review by asmafassy — COMMENTED** [review]
Overall you have done a great job @SeonaV  :blossom: I have left some comments to refactor and you can give it a try with "Template literals".  In time you may find it super easy and fun to use. You can find this article https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals for references. Good day :sun_with_face: !!

File: javascript/javascript1/week1/myjavascript.js (hunk 1 above)
Comment (SeonaV) [reply javascript/javascript1/week1/myjavascript.js]: ok..will change

File: javascript/javascript1/week1/myjavascript.js (hunk 3 above)
Comment (SeonaV) [reply javascript/javascript1/week1/myjavascript.js]: I have done the changes.

thanks

File: javascript/javascript1/week1/myjavascript.js (hunk 4 above)
Comment (SeonaV) [reply javascript/javascript1/week1/myjavascript.js:73]: done...

**Review by asmafassy — APPROVED** [review]
:+1: You can merge it now.

**Pull request merged**